docs/documentation/getting-started.md

```diff
@@ -98,193 +34,52 @@ accompany your sessions.
-        in focus. The color resembles the color of the active user.
-    - ![shared file](images/shared_file.png) A
-        blue arrow decorates a file that is shared with
-        other participants.
--   Cursors:
-    The position of a participant's cursor appears in the file in
-    his/her color.
--   Selections:
-    Any text selected by a participant also appears highlighted in
-    the file.
--   Changes:
-    If a participant writes something, his/her text will appear
-    highlighted in his/her color.
--   Locate participants:
-    On the right side of editors you can see colored bars representing
-    the viewports of each user. These show which part of the file each
-    user can see.
--   Follow participants:
-    Follow mode allows you to follow all movements of another user as
-    s/he moves within and between files. In the Session list, right
-    click on the user you wish to follow and select Follow Participant.
+If you want to configure Saros, open the `Saros` entry in the menu bar
+and click on `Preferences`.
 
 {% endcapture %}
-
-
 {% capture intellij %}
 
-**This page contains information partially specific to the current release `0.2.2`.<br>
-For information on previous releases, you can have a look at the corresponding [release notes](../releases/index.md).**
-
-The development of the Saros/I plugin is still in the alpha stages, so expect it to be a bit rough around the edges.
-There are still some [restrictions](#restrictions) that apply to the usage and some basic [features are still missing](#missing-features).
-
-## Disclaimer
-
-Saros/I does not include sub-modules when sharing a module (see [Module Restrictions](#module-restrictions)).
-As a consequence, such sub-modules might not be present for all session participants.
-If a participant deletes a shared directory that contains a sub-module in the local setup of another participant, this sub-module will be deleted without any notice.
-
-Furthermore, there are still some known bugs in the current release. Please have a look at the section [Known Bugs](#known-bugs).
-
-## Features
-
-This alpha version provides most of the basic functionality of Saros.
-You can
-
-- add existing XMPP-accounts
-- start a session with another person
-  - Sessions in Saros/I are currently limited to two participants (host and one client)
-- share exactly one module through Saros; the shared module must meet the restrictions described [here](#module-restrictions)
-- transfer the initial content of the module shared by the host to all participating clients
-- work on shared resources
-- create, delete, and move resources in the shared module
-- interact freely with non-shared resources
-- follow other participants of the session ([follow mode](features.md#follow-mode))
-
+Most interactions with the Saros session logic (like starting or ending a session) can be done through the Saros tool window.
```

Comment:
I still think there should be a note that Saros/I is currently still an alpha release and has specific restrictions/missing features. Linking to the latest release notes should be good enough instead of listing everything on this page.